database.example.yml

```diff
@@ -0,0 +1,32 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: postgres
+  host: localhost
+
+
+development:
+  <<: *default
+  database: circuitverse_development
+
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: circuitverse_test
+
+production:
+  username: <%= ENV['DB_USERNAME']%>
+  password: <%= ENV['DB_PASSWORD']%>
+  host: localhost
+  database: circuitverse_production
```

Comment:
Why is this file added?

Reply:
i deleted the database.example.yml file and added it before i deploy the file

Reply:
i added the database.example.yml because i deleted it by mistake